app/views/main/AllServicesOn.tsx

```diff
@@ -9,75 +9,47 @@ import {
 import { useTranslation } from 'react-i18next';
 import Pulse from 'react-native-pulse';
 import { SvgXml } from 'react-native-svg';
-import Geolocation from '@react-native-community/geolocation';
 
 import { Icons, Images } from '../../assets';
 import { Typography } from '../../components/Typography';
 
 import { styles } from './style';
-import { Colors } from '../../styles';
+import { Colors, Spacing } from '../../styles';
 import { Stacks, NavigationProp } from '../../navigation';
 import { useSelector, useDispatch } from 'react-redux';
-import isAutoSubscriptionEnabledSelector from '../../store/selectors/isAutoSubscriptionEnabledSelector';
-import getHealthcareAuthorities from '../../store/actions/healthcareAuthorities/getHealthcareAuthoritiesAction';
 import { RootState } from '../../store/types';
+import toggleAutoSubscriptionBanner from '../../store/actions/healthcareAuthorities/toggleAutoSubscriptionBannerAction';
 
 type AllServicesOnProps = {
   noHaAvailable: boolean;
```

Comment:
Can we remove this prop? I think it's unused and can be easily pulled from redux if needed. I would also ease the return logic on the `Main` screen.